ProxyAppsSuite/tests/miniamr.yaml

```diff
@@ -0,0 +1,895 @@
+_base:
+   summary: 'Apply a stencil calculation on a unit cube computational domain'
+   build:
+      summary: building the miniAMR test
+      modules: [ gcc, openmpi ]
+      env:
+         CC: mpicc
```

Comment:
add the following keys to the env section:
```
        env:
            http_proxy: https://proxyout.lanl.gov:8080
            https_proxy: https://proxyout.lanl.gov:8080
```
I know you can include this in your pavilion.yaml file, too, so it's not that critical.

Reply:
I added these lines to the environment but I changed the first line of
 ```http_proxy: https://proxyout.lanl.gov:8080```
to 
```http_proxy: http://proxyout.lanl.gov:8080```